src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Controllers/UserController.cs

```diff
@@ -65,5 +67,24 @@ public class UserController : HackathonControllerBase
             return Ok(userInfo);
         }
         #endregion
+
+        #region GetUploadToken
+        /// <summary>
+        /// Get file upload token
+        /// </summary>
+        /// <returns>The team</returns>
+        /// <response code="200">Success. The response return a token.</response>
+        [HttpPut]
+        [ProducesResponseType(typeof(Team), StatusCodes.Status200OK)]
+        [SwaggerErrorResponse(400, 404, 412)]
+        [Route("user/storageToken")]
+        [Authorize(Policy = AuthConstant.PolicyForSwagger.LoginUser)]
+        public async Task<object> GetUploadToken()
+        {
+            var token = UserManagement.GetUploadToken();
+            return Ok(token);

```

Comment:
We'd better not return the `token` as string, but return a `json` object. As a convention, we always return json objects.